### ISSUE TITLE
Issue with "Transparent" Textures

### ISSUE THREAD

**Issue by TheFlipsider** [Kerilk/noesis_bayonetta_pc#48]
Good morning/afternoon/evening!

So i've trying to export some models from bayonetta 3 (specifically of the homunculi), and basically all of them have textures that can only be visualized with the cycle blend feature. 
![example](https://github.com/Kerilk/noesis_bayonetta_pc/assets/145377997/6a0d25e9-6417-47bd-8210-972fd998336a)
No issue here. 
However, when exporting the model, these "transparent" textures are simply deleted in my experience.
![em2004001](https://github.com/Kerilk/noesis_bayonetta_pc/assets/145377997/a3cb025f-7194-48fd-bd83-5eea4906f43a)
![Captura de tela 2023-09-18 142139](https://github.com/Kerilk/noesis_bayonetta_pc/assets/145377997/643a05b1-a936-4a43-98aa-cc75cac9c89b)
I'm stumped on finding a solution to this problem. Is there any way to export the textures as if they had no blend, or to "fix" the textures after they are exported?
Either way, many thanks for your work, it helped a ton!

**Comment by TheFlipsider** [comment]
I figured it out! While messing around on GIMP, I simply turned off the "alpha" layer on the image and it worked. God, I feel so stupid right now. Feel free to delete this issue, or to keep it as help for anyone with a similar issue. 

For those using GIMP, simply decompose the texture, delete the alpha channel, and compose it. Presto! Ready to export.

Sorry for the mess, and many thanks again for the plugin!
